### ISSUE TITLE
Bootstrap fails "No such file" for llvm-objcopy

### ISSUE THREAD

**Issue by ehuss** [rust-lang/rust#132903]
Recently, build of rustc is failing with:

```
src.symlink_metadata() failed with No such file or directory (os error 2) ("src = /rust/build/aarch64-unknown-linux-gnu/stage1/lib/rustlib/aarch64-unknown-linux-gnu/bin/llvm-objcopy")
```

Steps:

1. `git clone https://github.com/rust-lang/rust.git`
2. `cd rust`
3. `./x.py build`

### Meta

Currently on d4822c2d84c242cc7403118b50c571464f38ef8f

Tested on aarch64-unknown-linux-gnu and x86_64-unknown-linux-gnu

**Comment by onur-ozkan** [comment]
cc @jieyouxu (I think you did some work related to this very recently, if I am not mistaken)

**Comment by jieyouxu** [comment]
Maybe duplicate of #132719?

**Comment by ehuss** [comment]
Yes, I verified that this is fixed by #132720, and that this was introduced by #131405.

**Comment by jieyouxu** [comment]
Thanks, closing as a duplicate of #132719.
